tuna/worker_interface.py

```diff
@@ -271,8 +276,11 @@ def get_job(self, find_state, set_state, imply_end):
                 else:
                   job.machine_id = self.machine.id
 
+                query = gen_update_query(job, self.job_attr,
+                                         self.dbt.job_table.__tablename__)
+                session.execute(query)
+
               session.commit()
```

Comment:
are we still committing anything here now that there is an execute() call? If not, please remove

Reply:
Executes must still be committed.